### PR TITLE
koord-scheduler: add quota debug restful api

### PR DESCRIPTION
**Pull request by buptcozy**
Signed-off-by: yangzhang <bupt_cozy@126.com>

### Ⅰ. Describe what this PR does

Add API to help diagnose and analyze ElasticQuota problems.

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/scheduler/plugins/elasticquota/plugin_service.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/koordinator-sh/koordinator/pkg/scheduler/frameworkext/services"
+)
+
+var _ services.APIServiceProvider = &Plugin{}
+
+func (p *Plugin) RegisterEndpoints(group *gin.RouterGroup) {
+	group.GET("/quota/:name", func(c *gin.Context) {
```
Comment (eahydra) [inline pkg/scheduler/plugins/elasticquota/plugin_service.go:30]: 这里是不是要支持namespace的.

File: pkg/scheduler/plugins/elasticquota/core/quota_summary.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+type QuotaInfoSummary struct {
+	Name              string `json:"name"`
+	ParentName        string `json:"parentName"`
+	IsParent          bool   `json:"isParent"`
+	RuntimeVersion    int64  `json:"runtimeVersion"`
+	AllowLentResource bool   `json:"allowLentResource"`
+
+	Max          v1.ResourceList `json:"max"`
+	OriginalMin  v1.ResourceList `json:"originalMin"`
```
Comment (eahydra) [inline pkg/scheduler/plugins/elasticquota/core/quota_summary.go:36]: OriginalMin是不是可以简化为Min?

File: pkg/scheduler/plugins/elasticquota/plugin_service.go (hunk 1 above)
Comment (buptcozy) [reply pkg/scheduler/plugins/elasticquota/plugin_service.go:30]: 经过讨论，暂时不改

**Review by eahydra — COMMENTED** [review]
/lgtm

**Review by eahydra — APPROVED** [review]
/lgtm

**Pull request merged**